storage-proofs/src/post/fallback/vanilla.rs

```diff
@@ -443,12 +444,10 @@ impl<'a, H: 'a + Hasher> ProofScheme<'a> for FallbackPoSt<'a, H> {
     fn satisfies_requirements(
         public_params: &Self::PublicParams,
         requirements: &Self::Requirements,
-        _partitions: usize,
+        partitions: usize,
     ) -> bool {
-        dbg!(&requirements);
-        dbg!(&public_params);
-        // Every partition must have the exact same number of challenges.
-        requirements.challenge_count == public_params.challenge_count
+        partitions * public_params.sector_count * public_params.challenge_count
+            >= requirements.challenge_count
```

Comment:
If we use `>=` here, then we *should* call it `minimum_challenges` (as in PoRep). If we call it `challenge_count` we should require that it be exactly `==`.

Reply:
fixed